src/scene.ts

```diff
@@ -4284,6 +4284,20 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         }
     }
 
+    private checkCameraRenderTarget(camera: Nullable<Camera>) {

```

Comment:
should it be checkCameraRenderTarget = (camera: Nullable<Camera>) => {} to prevent issue if called from callbacks ?